### PR TITLE
add some libraries, PHP extensions and frameworks.

### PR DESCRIPTION
**Pull request by peter279k**
Hi @pmjones , I add some frameworks and libraries. For example, Guzzle , Nette and Silex.

### CODE REVIEW

**Comment by sergeyklay** [comment]
@peter279k Could you please rebase

**Comment by peter279k** [comment]
Hi @sergeyklay , I rebase it now :).

**Comment by pmjones** [comment]
Hey @peter279k -- Some of the entries you list are not frameworks. If you can remove the non-framework ones (e.g. pecl, apc, memcache, and guzzle) I will review again and merge.

File: README.md [hunk 1]
```diff
@@ -149,3 +185,11 @@ These are the years-of-introduction of various frameworks and library collection
 - Phalcon 3.0.0: <https://blog.phalconphp.com/post/phalcon-3-0-0-released>
 
 - Zend Framework 3: <https://framework.zend.com/blog/2016-06-28-zend-framework-3.html>
+
+- Phalcon 2: <https://github.com/phalcon/cphalcon/blob/master/CHANGELOG.md>
```
Comment (sergeyklay) [inline README.md]: Wrong, 2015

File: README.md [hunk 2]
```diff
@@ -149,3 +185,11 @@ These are the years-of-introduction of various frameworks and library collection
...
+
+- PHPixie 3: <https://github.com/PHPixie/Framework/releases>
+
+- Phalcon 3: <https://github.com/phalcon/cphalcon/blob/master/CHANGELOG.md>
```
Comment (sergeyklay) [inline README.md:161]: Duplicate

File: README.md [hunk 3]
```diff
@@ -132,6 +154,20 @@ These are the years-of-introduction of various frameworks and library collection
...
+
+- Guzzle 4: <https://github.com/guzzle/guzzle/tree/4.0.0>
+
+- Guzzle 5: <https://github.com/guzzle/guzzle/tree/5.0.0>
+
+- Guzzle 6: <https://github.com/guzzle/guzzle/tree/6.0.0>
+
+- PHPixie 2: <https://phpixie.com/blog/new-phpixie-database-and-configuration-libraries.html>
+
+- Phalcon 1: <https://github.com/phalcon/cphalcon/blob/master/CHANGELOG.md>
```
Comment (sergeyklay) [inline README.md]: Duplicate + wrong

File: README.md [hunk 4]
```diff
@@ -132,6 +154,20 @@ These are the years-of-introduction of various frameworks and library collection
 
 - Yii 2: <http://www.yiiframework.com/news/81/yii-2-0-0-is-released/>
 
+- Phalcon 1: <https://github.com/phalcon/cphalcon/blob/master/CHANGELOG.md>
```
Comment (sergeyklay) [inline README.md]: Duplicate + wrong

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by peter279k** [comment]
Thanks for the review. I remove the duplicated entries and remove the non framework entries.

**Comment by pmjones** [comment]
Sorry, last change -- PEAR 1 is a "library collection" so it stays. That's the only issue left; once it gets added back I can merge. Thanks again!

**Pull request merged**